controllers/hyperconverged/hyperconverged_controller.go

```diff
@@ -80,13 +80,24 @@ const (
 	hcoVersionName    = "operator"
 	secondaryCRPrefix = "hco-controlled-cr-"
 	apiServerCRPrefix = "api-server-cr-"
+	configMapCRPrefix = "cm-policy-cr-"
 
 	// These group are no longer supported. Use these constants to remove unused resources
 	v2vGroup = "v2v.kubevirt.io"
 
 	requestedStatusKey = "requested status"
 )
 
+// triggeredBy informs which resource has triggered the Reconcile loop
+type triggeredBy int8
+
+const (
+	triggeredByHco triggeredBy = iota
+	triggeredByApi
+	triggeredByCm
```

Comment:
After reading the code again, this constant is only used for the rate limit CM. I think this name is confusing. Could you please rename? e.g. `triggeredByRateLimitCM` or something like this?

Reply:
I have used `triggeredByStaticRateLimitCM`. I find it more self-explanatory.